dev/TabView/TabView.h

```diff
@@ -14,6 +14,9 @@
 #include "TabViewTabDragCompletedEventArgs.g.h"
 #include "DispatcherHelper.h"
 
+static constexpr double c_tabShadowDepth = 16.0;
```

Comment:
Looks like you are reading the value from theme resource and this is no longer being used?

Reply:
That's the default value in case it can't be read from resource for some reason.